### DISCUSSION TITLE
Error with Defaults versus YAML Header

### DISCUSSION THREAD

**Discussion by dstark** [jgm/pandoc#10214] [Q&A] [answered]
When I use, for example, `abstract-title` or `reference-section-title` in a YAML header (and not in a defaults file), those titles render fine in a document based on other settings in a defaults file. If, however, I try to include `abstract-title: Abstract` or `reference-section-title: Bibliography` in that defaults file (and not in the document's own YAML header), I get an `unknown option` error for `abstract-title` or `reference-section-title`.

What am I missing as far as the process for including these options in a defaults file? Or do they (and others?) specifically need to be set in a document header (or in the pandoc command?) rather than in a defaults file?

Thanks so much for the clarification!

**Comment by twsh** [comment, marked as answer]
You can include them in your defaults file as metadata:

```
metadata:
  abstract-title: Foo
  reference-section-title: Bar
```

**Reply by dstark** [reply]
Ah, yes. That's what I was missing. Makes perfect sense out of the command-line argument version that I'd seen but just hadn't translated to how that should look in the defaults file. Thanks so much!
